app/build.gradle

```diff
@@ -0,0 +1,29 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.3"
+
+    defaultConfig {
+        applicationId "com.yoyo.jopet.androidorgapp"
+        minSdkVersion 21
```

Comment:
Minimum version is rather new (Lollipop and above) which means no love for KitKat and below... :crying_cat_face: